style.css

```diff
@@ -0,0 +1,718 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
+
+.nav-container a {
+  display: flex;
+  align-items: center;
+  margin-left: 20px;
+  flex: 1;
+  height: 100%;
+  color: white;
+  text-decoration: none;
+  list-style: none;
+}
```

Comment:
- [ ] Kindly make sure your logo text typography looks exactly as much as possible to the Figma design.

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212620761-c72d6932-0255-4148-97f5-493ea2c2add5.png)

   - ### FIGMA DESIGN
     
      ![image](https://user-images.githubusercontent.com/87197412/212621022-6ad25a7c-147b-42e0-82bc-dd2964c9e999.png)